### DISCUSSION TITLE
Localization coordinates

### DISCUSSION THREAD

**Discussion by Anudeepsrib** [alzayats/DeepFish#8] [Q&A] [answered]
Hi, I am trying to use the pre-trained model (trained on the deep fish dataset) to make predictions on an external dataset. 
Can you please suggest how to get the (x,y) coordinates in the output?

**Comment by IssamLaradji** [comment, marked as answer]
Thanks for your interest, you could try the blobs2points function in this repo: https://github.com/ServiceNow/LCFCN

**Reply by Anudeepsrib** [reply]
Thank you @IssamLaradji. I was able to get the coordinates 👍
